torch/_dynamo/variables/base.py

```diff
@@ -222,7 +222,7 @@ def num_parameters(self):
         unimplemented(f"num_parameters: {self}")
 
     def call_hasattr(self, tx, name: str) -> "VariableTracker":
-        unimplemented(f"hasattr: {self}")
+        unimplemented(f"hasattr: {repr(self)}")
```

Comment:
Do you want to apply this consistently to the other unimplemented in this file too? E.g., num_parameters above

Reply:
I only applied it to the ones I found in debugging graph breaks. But I can apply it everywhere, sure.